src/crypter.js

```diff
@@ -66,6 +71,7 @@ module.exports.Crypter = (secret: Buffer, {
       }
 
       const rawEncrypted = fromUrlSafeBase64(base64);
+      // a diff in ivlength between the payload & `ivLength` would break this
```

Comment:
You mean you can't change the ivLength arbitrarily? They usually have a static length requirement depending on the algorithm used. So this is not something one would expect to ever happen (without using completely different tokens).

Reply:
I'm just confirming that if there is somewhere where a different length is used, it would fail in a way that's non-obvious. ie. failure to decrypt can be mismatch between iv in token vs. configuration of this object, but it would fail in a generic unable to decrypt way and there's no way to really provide any other debug info on that (unless the error from Crypto is able to make a distinction...?)

Reply:
Actually, reading the above, is specifying ivLength as a param even necessary if the majority of them are required based on algo..?

Also - does initialization of a particular algo fail if an insufficient iv is provided?

Reply:
yeah - you *could* remove the iv arg entirely, and specify the cypher algo from a constant list that also always sets the correct config internally. Erring more on the side of "very hard to accidentally misconfigure" by removing some possibilities, however unlikely.

Reply:
I would expect that customising the config rarely happens. If someone decides to change it they should to a good investigation on the algorithms and their advantages. So if someone really want's to mess this up they can.